### DISCUSSION TITLE
Unity WebGl Android and iOS Mobile Browser.

### DISCUSSION THREAD

**Discussion by Androser** [Web3Auth/Web3Auth#445] [Help]
Hello! I've been looking for a way to connect a Torus wallet or any other with Web3Auth login authentication, and even though the authentication already works... there is no way to make transactions, maybe I missed some information, but I can't find any luck. I would like to ask for your help and if someone has the answer, please share it

**Comment by himanshuchawla009** [comment]
@Androser , @yashovardhan  is writing a demo app to demonstrate transactions with unity sdk, he will get back to you once we will publish the docs and example for the same. Expect it till next week.

**Comment by yashovardhan** [comment]
Hey @Androser,

Sorry for the delay in getting back to you here. We recently published our Unity SDK Reference, Integration Builder and Guide. We have mentioned all the ways you can install, initialiise and use our SDK. Let me know if you have any further questions on this. I'm providing the links for it here:

- Unity SDK Reference: https://web3auth.io/docs/sdk/unity/
- Unity Implementation Guide: https://web3auth.io/docs/guides/unity
- Unity Sample App Integration Builder: https://web3auth.io/docs/integration-builder?lang=unity&chain=eth&customAuthentication=no&whitelabel=no&mfaParams=no&usingEmailPasswordless=no&rnWorkflowMode=expo&evmFramework=web3
- Unity Example Application: https://github.com/Web3Auth/examples/tree/main/unity

**Comment by Androser** [comment]
Thanks, I'll check it out. But do it works on Unity WebGl? You have do a
great job. Thanks!!

El lun, 5 sept 2022 a la(s) 16:36, Yashovardhan Agrawal (
***@***.***) escribió:

> Hey @Androser <https://github.com/Androser>,
>
> Sorry for the delay in getting back to you here. We recently published our
> Unity SDK Reference, Integration Builder and Guide. We have mentioned all
> the ways you can install, initialiise and use our SDK. Let me know if you
> have any further questions on this. I'm providing the links for it here:
>
> Unity SDK Reference: https://web3auth.io/docs/sdk/unity/
> Unity Implementation Guide: https://web3auth.io/docs/guides/unity
> Unity Sample App Integration Builder:
> https://web3auth.io/docs/integration-builder?lang=unity&chain=eth&customAuthentication=no&whitelabel=no&mfaParams=no&usingEmailPasswordless=no&rnWorkflowMode=expo&evmFramework=web3
> Unity Example Application:
> https://github.com/Web3Auth/examples/tree/main/unity
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Web3Auth/discussions/445#discussioncomment-3564193>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOOD4V4KPY3RWM7J7GDWHZ3V4ZRWHANCNFSM57DL5WKQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

**Click en los iconos
<https://api.whatsapp.com/send/?phone=573219267844&text=Hola%20Andr%C3%A9s%2C+*buen+d%C3%ADa*+vengo+gmail;+le+di+click+a+tu+firma+de+Gmail+*andresleonardogomez%40gmail.com*&app_absent=0>
<https://www.facebook.com/profile.php?id=100022532341789>
<https://www.instagram.com/androser13139/?hl=es-la> para ir a la página**




Andrés L. Gómez Gil

Tecnólogo Diseño e Integración de Automatismos Mecatrónicos

@Leonardo_Gil

@Androser1313

www.educapture.edu.co

<https://api.whatsapp.com/send/?phone=573219267844&text=Hola%20Andr%C3%A9s%2C+*buen+d%C3%ADa*+vengo+gmail;+le+di+click+a+tu+firma+de+Gmail+*andresleonardogomez%40gmail.com*&app_absent=0>
<https://www.facebook.com/profile.php?id=100022532341789>
<https://www.instagram.com/androser13139/?hl=es-la>

**Reply by Oberhaim** [reply]
did you have an answer for this ? 
Thanks!
